pyi.Dockerfile

```diff
@@ -80,6 +82,23 @@ WORKDIR /tmp
 COPY scripts/burn_build.sh .
 RUN ./burn_build.sh
 
+# node-package-builder-musl
+FROM alpine${NODE_PACKAGE_BUILDER_MUSL} AS node-package-builder-musl
+WORKDIR /tmp
+RUN apk add --no-cache curl g++ python3 make gcc git bash nodejs npm
```

Comment:
If you find yourself repeating these - prefer to put them in a script `node_builder_musl_env.sh` that's used in `Dockerfile` and `pyi.Dockerfile`